bench/bench.ml

```diff
@@ -0,0 +1,83 @@
+open Bechamel
+open Toolkit
+open Ocamlformat_lib
+
+type range = int * int
+
+type input =
+  { name: string
+  ; input_name: string
+  ; kind: Syntax.t
+  ; source: string
+  ; conf: Conf.t
+  ; action: [`Format | `Numeric of range] }
+
+let source_from_file f =
+  Stdio.In_channel.with_file f ~f:Stdio.In_channel.input_all
+
+let inputs =
+  let source_ml = source_from_file "test/passing/tests/source.ml" in
```

Comment:
this means that your benchmark will move everytime you add things to that file. that can make it difficult to track what's going on across versions.

Reply:
also is you read that file at runtime make sure to register it as a dependency otherwise the benchmark won't get reexecuted and things like `dune clean; dune build @runbench` won't work

Reply:
done